include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -31,117 +31,123 @@
 // DAMAGE.
 //
 // ==========================================================================
-// Author: Hannes Hauswedell <hannes.hauswedell@fu-berlin.de>
-// Author: Marcel Ehrhardt <marcel.ehrhardt@fu-berlin.de>
-// Author: David Heller <david.heller@fu-berlin.de>
-// ==========================================================================
+
+/*!\file
```

Comment:
It produces the same as `second (and likely the first)`, see this image

![doxygen_file_empty_output2](https://cloud.githubusercontent.com/assets/103778/26095225/2c29de04-3a1d-11e7-9815-a02b4ab775e5.png)

I only copied `union_alphabet.hpp` to `xyz/union_alphabet.hpp` and renamned in the file `union_alphabet` with `union_alphabet2`. And `\file` without filename did the right job.